### ISSUE TITLE
stopPropagation errors logs in useSelect

### ISSUE THREAD

**Issue by hayes** [adobe/react-spectrum#2263]
# 🐛 Bug Report 

Using keyboard to open select built with useSelect logs an error to the console

## 🤔 Expected Behavior

No errors should be logged to the console

## 😯 Current Behavior

The following error is logged:

stopPropagation is now the default behavior for events in React Spectrum. You can use continuePropagation() to revert this behavior.

## 🔦 Context

Causes noise in console, and breaks our testing flow which fails when errors are logged to the console.

## 💻 Code Sample

This can be reproduced by using keyboard to open a select in the official docs (arrow keys, space, and enter all trigger the error): https://react-spectrum.adobe.com/react-aria/useSelect.html
## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| react-spectrum   |
| Browser          |
| Operating System |

**Comment by snowystinger** [comment]
Yep, I see that, thanks for reporting! It looks like it's from https://github.com/adobe/react-spectrum/blob/0ae06f525d6bf5593c99b86fc913f5f06e6d46c7/packages/%40react-aria/menu/src/useMenuTrigger.ts#L59
We should probably check if propagation has already been stopped or just remove those two lines
I think useFocusable is responsible for applying the error

**Comment by Anuragtech02** [comment]
Hey @snowystinger , I think I can give this a try! Could you please assign this to me?
